documentation/modules/rn-2.2.adoc

```diff
@@ -40,6 +52,15 @@ As a temporary workaround, the `Importer` pod is removed at the end of the preco
 
 This issue only affects the `Importer` pod log and warm migration. Cold migration and the `virt-v2v` logs are not affected.
 
+.Unclear error message for failed migration of VMs with no operation system
+
+When migration plans for VMs with no operation system fail, the error message does not indicate the reason for the failure. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2008846[*BZ#2008846*])
```

Comment:
Change to "Error status message for a VM with no operating system on the *Migration plan details* page of the web console does not describe the reason for the failure. (link....)"